src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import arrow from 'components/LoginPage/images/arrow.svg'
+import slideImage1 from 'components/LoginPage/images/slide-3_img-1.svg';
+import slideImage2 from 'components/LoginPage/images/slide-3_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 0%;
+  text-align: center;
+  .note-example {
+    width: 61%;
+  }
+   .edit-example {
+    position: relative;
+    margin-top: -9%;
+  }
+  .bottomPosition {
+    margin-top: -65%;
+    margin-bottom: 59%;
+    position: relative;
+  }
+  .bottomTooltip {
+    width: 22%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+  }
+  .bottomHeader {
+    margin-top: -53%;
+    margin-bottom: 45%;
+    position: relative;
+  }
+  .shadow {
+  -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+  filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+}
+`;
+const H2 = styled.h2`
+  padding-bottom: 7%;
+  font-size: calc(16px + (30 - 16)*(100vw - 375px)/(1200 - 375));
+  text-align: center;
+  color: #ffffff;
+`;
+const H3 = styled.h3`
+  padding-bottom: 7%;
+  text-align: center;
+  color: #ffffff;
+`;
+const Figure = styled.figure`
+  position: relative;
+  width: 74%;
+  margin: auto;
+  height: auto;
+`;
+const Img = styled.img`
+  width: 100%;
+  height: auto;
+`;
+const ArrowUp = styled.img`
+  width: 2%;
+  height: auto;
+`;
+const ArrowDown = styled.img`
+  width: 2%;
+  height: auto;
+  transform: rotate(180deg);
+`;
+const ArrowLeft = styled.img`
+  width: 1.5%;
+  height: auto;
+  transform: rotate(270deg);
+  position: absolute;
+  left: 36%;
+  top: 38%;
+`;
+const ArrowRight = styled.img`
+  width: 1.5%;
+  height: auto;
+  transform: rotate(90deg);
+  position: absolute;
+  right: 36%;
+  top: 38%;
+`;
+const Tooltip = styled.div`
```

Comment:
`Tooltip` is declared in `LoginPageComponent.js` and in `Slide3.js` with almost the same styles.

Reply:
And it's not a tooltip: a tooltip is a "hover box" that appears when user points a cursor over an element.
This is just a text box that is always displayed, irrelevant to the cursor position, so it's not a tooltip.
I call this things "hints": they provide additional info for the user on the picture.

<img width="791" alt="tooltip" src="https://user-images.githubusercontent.com/6686679/69766464-c366c600-1180-11ea-8239-3dfb6d641e29.png">

Reply:
Thanks! That sounds realy good! Sorry, that I could not to find appropriative word.

Reply:
First part fixed in commit [03c5f6d](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/03c5f6d943fc184d59ab4625f6129eb16746f55a) and second in [661b2d7](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/661b2d774c1c591c5624f53d6015c4d90343adc1)